packages/completer/style/base.css

```diff
@@ -329,7 +329,7 @@
   display: none;
 }
 
-[data-command='inline-completer:next'] > .jp-ToolbarButtonComponent-icon,
-[data-command='inline-completer:previous'] > .jp-ToolbarButtonComponent-icon {
+.jp-InlineCompleter [data-command='inline-completer:next'] svg,
+.jp-InlineCompleter [data-command='inline-completer:previous'] svg {
```

Comment:
Can you please avoid bare `svg` selectors? It has a sever performance penalty when notebooks include many svg visualisations. The selector needs to rewritten to use child combinator (`>`) or a class name instead.

Reply:
Fixed in https://github.com/jupyterlab/jupyterlab/pull/15021/commits/31d2e47b92a0b72a6a440c1cd8c9c232ac43d8e7